specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -508,7 +508,14 @@ model RegionalConfiguration {
   @doc("""
       The number of engine instances to execute load test in specified region. Supported values are in range of 1-400.
     """)
-  engineInstances: int32;
+  @madeOptional(APIVersions.v2025_03_01_preview)
```

Comment:
This is not needed imo? This was always optional ?

Reply:
Yeah we can remove this (to avoid confusion), the field has always been optional. Can also see that there is no difference between the new swagger and the old

Reply:
The one in regionalConfiguration was not optional